showcase/app/templates/components/advanced-table.hbs

```diff
@@ -8,18 +8,6 @@
 <Shw::Text::H1>AdvancedTable</Shw::Text::H1>
 <section data-test-percy>
 
-  <Shw::Text::Body>
-    <a
```

Comment:
I think it would be useful to either leave the link at the top of the page or move the demo up higher. Designers mainly use the demo for their testing.